includes/Checker/Abstract_Check_Runner.php

```diff
@@ -156,6 +164,17 @@ final public function set_plugin( $plugin ) {
 		$this->plugin = $plugin;
 	}
 
+	/**
+	 * Sets whether to include experimental checks in the process.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param bool $include_experimental True to include experimental checks. False to exclude.
+	 */
+	final public function set_experimental_flag( $include_experimental ) {
+		$this->include_experimental = $include_experimental;
+	}
```

Comment:
The approach of having a method like this doesn't work I think, for the same reason that we had to introduce the abstract `get_check_slugs_param()` method. The way it's implemented now, the parameter wouldn't be picked up if the class is initialized early, in the `object-cache.php` drop-in.

So I think we need to instead go with an abstract method like `get_include_experimental_param() : bool`, which would need to be implemented for both the CLI and AJAX runner (for the latter we could just return `false` for now as there's no support yet for this in WP Admin / AJAX).

Reply:
Thanks @felixarntz I've updated this to work similar to the other params.